src/cortxfscli/cortxfscli.py

```diff
@@ -167,14 +177,17 @@ def add_args(parser):
 		sbparser.add_argument('action', help='action', choices=['create', 'list', 'delete'])
 		sbparser.add_argument('args', nargs='*', default=[], help='fs command options')
 		sbparser.set_defaults(command=FsCommand)
+		sbparser.add_argument('-cv', '--config_validation',
+			help='Optional JSON config file for validation rules.',\
+			default=cortxfscli_validation_rule_dev)
 
 	def validate_args_payload(self, args):
 		if args.action.lower() == 'list' and len(args.args) != 0:
 			throw_exception_with_msg("Too many args for " + \
 			args.action.lower())
 
 		if args.action.lower() == 'create' or args.action.lower() == 'delete':
-			if len(args.args) != 1:
+			if len(args.args) != 2:
```

Comment:
why we need bsize for delete operation ?
And if bsize is not provided, then we should be able to use the default one instead of mandating on providing bsize

Reply:
thanks for catching this, this is a bug.

Reply:
Fixed this issue, also addressed the new min and max values for valid bsizes as discussed following the motr's values.